scripts/dashboard-version

```diff
@@ -1,3 +1,4 @@
 # This file is updated by running scripts/update-dashboard.sh
 # Don't edit it manullay
+# the version will become X.Y.Z-<commit-short-hash> format later
```

Comment:
It's better support origin tag value, such as:
`v7.6.0-f7bbcdcf-test`.

Reply:
This file is also used in `Makefile` and `embed-dashboard-ui.sh` to describe the dashboard version without `v` prefix as before, so that why I remove the `v` prefix here, else we need to modify the `Makefile` and `embed-dashboard-ui.sh` logic.